src/azul/tdr.py

```diff
@@ -183,3 +186,9 @@ def _get_source_info(self, source: TDRSource) -> JSON:
 
     def _repository_endpoint(self, path_suffix: str):
         return f'{config.tdr_service_url}/api/repository/v1/{path_suffix}'
+
+
+class AuthorizedDRSClient(DRSClient, SAMClient):
+
+    def __init__(self):
+        super().__init__(self.oauthed_http)
```

Comment:
Not much better than the original approach because it still goes against the spirit of a comment from my previous review.

> Hacky. I shouldn't have to instantiate a SAMClient only to steal one of its dependencies.

Yes, technically you're only creating one instance but the cost of creating it now includes the cost of instantiating both parent classes.

But don't worry about this for now. I might refactor it once we work out the other issues.